src/main/java/util/CIDevelopLoaderImpl.java

```diff
@@ -14,6 +14,26 @@ public String getBasePathUrl() {
         }
     }
 
+    @Override
+    public String getApiKeys() {
+        String variable = System.getProperty("DEVELOP_BASE_PATH");
```

Comment:
This may require some additional pipeline change to check with Tanio. I could see the following error in the tests so this may be the case of setting this up in the pipeline

Automation exception: Setup of variable DEVELOP_BASE_PATH failed, please pass in command or configure variable

Reply:
this class no longer exist